File: CS/15_Document/IsPDFPortfolio.cs
using Spire.Pdf;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace IsPDFPortfolio
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create a pdf document
            PdfDocument doc = new PdfDocument();

            //Load from file
            doc.LoadFromFile(@"..\..\..\..\..\..\Data\TextBoxSampleB_1.pdf");

            //Judge whether the document is portfolio or not.
            bool value = doc.IsPortfolio;
            if (value)
            {
                MessageBox.Show("The document is portfolio");
            }
            else
            {
                MessageBox.Show("The document is not portfolio");
            }
        }
    }
}

File: VB/15_Document/IsPDFPortfolio.vb
Imports Spire.Pdf

Namespace IsPDFPortfolio
    Partial Public Class Form1
        Inherits Form
        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
            ' Create a new PdfDocument object
            Dim doc As New PdfDocument()

            ' Load the PDF document from the specified file path
            doc.LoadFromFile("..\..\..\..\..\..\Data\TextBoxSampleB_1.pdf")

            ' Check if the document is a portfolio
            Dim value As Boolean = doc.IsPortfolio

            ' Display a message box indicating whether the document is a portfolio or not
            If value Then
                MessageBox.Show("The document is a portfolio")
            Else
                MessageBox.Show("The document is not a portfolio")
            End If

            ' Close the PDF document
            doc.Close()
        End Sub
    End Class
End Namespace
